packages/common/src/reservation-form/ReservationFormField.tsx

```diff
@@ -445,7 +445,8 @@ const ReservationFormField = ({
           defaultValue={defaultValue ? String(defaultValue) : undefined}
           invalid={!!error}
           required={required}
-          maxLength={MAX_TEXT_LENGTH}
+          // email field is special and has one less character than the rest
+          maxLength={MAX_TEXT_LENGTH - (isEmailField ? 1 : 0)}
```

Comment:
Why should email fields have one less character? 🤔

Reply:
Good question, something weird in the backend constraints. 255 for all other fields, 254 for email 🙃  
The GQL field error specifies the max length.